### ISSUE TITLE
not changing settings properly

### ISSUE THREAD

**Issue by cryptojohnny81** [Legedric/ptmagic#145]
PTMagic won't set settings to PT.  Triggers assigned for readyforliftoff are met and it says in PT that settings are for readyforlightoff but actual settings remain for bear settings.  Not sure why this is happening and I also for the life of me can't get my monitor to work. Running windows.  I think PTMagic will be a great add on once I can get theses issues hashed out.  Please help. Thanks

**Comment by Legedric** [comment]
Pleae post screenshots of your issues, they help for us to find the actual issue.

**Comment by cryptojohnny81** [comment]
Thank you for your response.  I will try and screenshot next time this happens but I was able to get the monitor to work.  Still have the issue when the market turns and switches strategies the setting will not take effect.  For example today when the market was on an uptrend and should have been set to tothemoon it was still showing bearmarket settings.  I installed the new update and hope this may fix the problem.  Also when I change a setting while running PTMagic it will not implement the adjustment.  Example being I changed dca value for the tothemoon and readyforliftoff while running and in tothemoon settings which had it at 1.3, I made the adjustment to 0.75, and it never took affect.  I had to manually change it.  Thank you for you help
